src/lib/components/index/Season.svelte

```diff
@@ -16,19 +16,26 @@
 	// Update visuals (background, video, player/team buttons, tables)
 	// Get color for teams to work (might need client loading?, use await?)
 
+	interface FantasyTeamWithPlayers extends Tables<'fantasy_teams'> {
```

Comment:
Should we have these extended types be stored somewhere in the types folder?

Reply:
Yeah I think so, but maybe when we are a bit more "certain" what the types need to actually be? Cause now its kinda up in the air idk